tools/certa/src/CertaConfig.ts

```diff
@@ -69,6 +69,7 @@ export interface CertaConfig {
    * The absolute path to a JavaScript module containing backend initialization logic.
    * This module will be required in Certa's "main" process ***before*** initializing a test runner.
    * If the file's `module.exports` is a Promise, that Promise will also be awaited before running tests.
+   * The exported Promise may also optionally be resolved with a "cleanup" async callback, which will be executed _after_ running tests.
```

Comment:
Not sure if it's clear from this line that it need to be returned as part of the init() function.

Reply:
There's no `init` function as far as certa is concerned - just a CommonJS module w/ a promise as it's `module.exports`

Typically we set `module.exports` to the _return value_ of some `init` function, but that's an implementation detail on the test side.